linux_pipeline/Jenkinsfile_lis_rpm_hyperv

```diff
@@ -29,28 +29,31 @@ properties ([
 ])
```

Comment:
add to commit message the area for the test, in this case you can do something like:

lis_rpm: Added support for RHEL 6.10